manager.go

```diff
@@ -111,15 +114,11 @@ func (m *Manager) jsonRequest(subj string, req interface{}, response interface{}
 
 	jsr, ok := response.(jetStreamResponseError)
 	if !ok {
-		return nil
-	}
-
-	if jsr.ToError() != nil {
 		return jsr.ToError()
 	}
 
-	if m.validator == nil {
-		return nil
+	if m.validator == nil || true {
```

Comment:
Whats the purpose of the `|| true` here, it disables all request validation doesnt it?

Reply:
> The only thing I had to bypass here was the json schema check.

I had to bypass the validation as the schema requires either: **just an error or just data.** 
I looked at the condition and what would be needed is adding another clause of the form: error with code 10004 then data is ok.
However, I'm not that deep into json schema. That is what I'd need your help for

Reply:
Ah, right, I had not put 1+1 together ont hat.  That is not a good thing, I'd rather broken streams just not show and the error indicates which one then.  Not comfortable changing things so that error+data is present at the same time at this stage.

Reply:
What If I added a list of missing streams?

Reply:
To io.nats.jetstream.api.v1.stream_list_response? I see that schema is actually invalid as it is - consumers key need to be streams key, but that would be ok to add there yes

Reply:
ok, I'll add that to the server then. 
In that case we will have error with code 10004 + data + missing list

Reply:
We cannot have error+data

Reply:
Then I will remove the error code?

Reply:
Yes, essentially we're saying for this api call missing responses is not an error, but we provide hints on whats missing.

We can then report that nicely in the cli and it can be a list of all missing that way.

Reply:
added: https://github.com/nats-io/nats-server/pull/2739